src/contexts/useAuth.jsx

```diff
@@ -1,42 +1,40 @@
-import React, {createContext, useContext, useEffect, useState} from 'react'
-import {auth} from '../auth/firebase.jsx'
-import {useUser} from "./useUser";
+import React, { createContext, useContext, useEffect, useState } from 'react';
+import { auth } from '../auth/firebase.jsx';
+import { useUser } from './useUser';
 
-const AuthContext = createContext({})
+const AuthContext = createContext({});
```

Comment:
File indentation has been changed for this and some other files 
Please hit cmd + option + L on a jetbrains software (pycharm) to indent code before committing

Reply:
Not file indentation. Since we are using semi-colons I think it is consistent to have it here. We should have semi-colons every where or no where, and avoid doing it sometimes.